hadoop-ozone/dist/src/main/smoketest/admincli/cert.robot

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Smoketest ozone cluster startup
+Library             OperatingSystem
+Library             BuiltIn
+Resource            ../commonlib.robot
+Test Timeout        5 minutes
+
+*** Variables ***
+
+*** Test Cases ***
+Run cert list
+    ${output} =         Execute          ozone admin cert list
+    Should Contain      ${output}        Certificate list:(Type=
+
+cert list as JSON
+    ${output} =         Execute             ozone admin cert list --json 2>> certInfo
+    ${errOutput} =      Execute             cat certInfo
+                        Execute             echo '${output}' | jq -r '.'
+    Should Contain      ${errOutput}        Certificate list:(Type=
+                        Execute             rm certInfo
```

Comment:
I think this works only if security is enabled, so we need something like this:

https://github.com/apache/ozone/blob/f182b81751532ed5fa3b15aac7e6061a9f7f73d0/hadoop-ozone/dist/src/main/smoketest/basic/ozone-shell-lib.robot#L213

as the test is [failing](https://github.com/Tejaskriya/ozone/actions/runs/5572469036/jobs/10179226421#step:5:679) in unsecure:

```
Run cert list                                                         | FAIL |
255 != 0
------------------------------------------------------------------------------
cert list as JSON                                                     | FAIL |
255 != 0
```